### ISSUE TITLE
[Dark Mode] The color contrast of selected list Item's Back color and Border color is insufficient

### ISSUE THREAD

**Issue by Amy-Li03** [dotnet/winforms#13778]
### .NET version

.NET 10 SDK build: 10.0.100-rc.1.25401.103

### Did it work in .NET Framework?

Not tested/verified

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

Occurs when DarkMode feature involved

### Issue description

When the application is in Dark Mode, the color contrast of selected list Item's Back color and Border color is insufficient.

<img width="1115" height="788" alt="Image" src="https://github.com/user-attachments/assets/c035a3fa-d0e4-4bfa-832b-a841cdc08f56" />

### Steps to reproduce

1. Create a WinForms .NET Core application with ListView control
2. Add some items for listview control
3. Enable DarkMode as below  in Program.cs file
```
#pragma warning disable WFO5001
      Application.SetColorMode(SystemColorMode.Dark);
#pragma warning restore WFO5001
```
4. Build and run the application
5. Selected the list item, and then using the Accessibility Insight tool to observe the color contrast between selected list Item's Back color and Border color

**Comment by merriemcgaw** [comment]
Dark mode can't be fully accessibility compliant for contrast - users are going to want to use the High Contrast modes. We will consider this not a bug.
